dockers/docker-database/database_config.json.j2

```diff
@@ -24,6 +24,12 @@
             "persistence_for_warm_boot" : "yes"
         }
 {% endif %}
+        ,"redis_bmp":{
+            "hostname" : "redis_bmp.server",
+            "port": 6399,
```

Comment:
>6399 [](http://example.com/codeflow?start=20&length=4)

Magic number.
Actually I see the value of REMOTE_DB_PORT named constant. It help organized the port allocation in a central place, easy to detect port confliction. Could you follow?

Reply:
remove port since we only have local connection requirement, thus domain socket is fine.